### ISSUE TITLE
Missing VUID for multiview on render pass creation

### ISSUE THREAD

**Issue by Rua** [KhronosGroup/Vulkan-Docs#1774]
The documentation for the `multiview` feature states:
> If this feature is not enabled, the view mask of each subpass **must** always be zero.

There is currently no VUID stating this requirement for the `viewMask` members of `VkRenderPassMultiviewCreateInfo` and `VkSubpassDescription2`. VUIDs for the `multiviewGeometryShader` and `multiviewTessellationShader` features may also be missing at those locations.

**Comment by jekstrand** [comment]
The spec does have the following:

> VUID-VkRenderPassCreateInfo2-viewMask-03058
> The [VkSubpassDescription2](https://www.khronos.org/registry/vulkan/specs/1.3-extensions/html/vkspec.html#VkSubpassDescription2)::viewMask member of all elements of pSubpasses must either all be 0, or all not be 0

Does that cover your case?  Or are you talking about trying to combine `VkRenderPassMultiviewCreateInfo` and `vkCreateRenderPass2()`?  If the later, I don't think that's allowed.

**Comment by Rua** [comment]
I'm referring to the feature requirement. There is no VUID stating that the `viewMask` must all be 0 if the `multiview` feature hasn't been enabled - or conversely, that if the `viewMask` are all not 0, that the `multiview` feature must be enabled.

**Comment by janharaldfredriksen-arm** [comment]
I agree we're missing something for the multiview feature requirement.

VUID-VkGraphicsPipelineCreateInfo-renderPass-06047 and VUID-VkGraphicsPipelineCreateInfo-renderPass-06048 should cover the `multiviewGeometryShader` and `multiviewTessellationShader` cases.

Dynamic rendering added some VUs like this:
```
VUID-VkRenderingInfo-multiview-06127
If the multiview feature is not enabled, viewMask must be 0
```
But we don't seem to have a corresponding VU for non-dynamic rendering.

**Comment by oddhack** [comment]
This should be fixed in the 1.3.207 spec update.
